### DISCUSSION TITLE
How to embed slint rust into the qt window？

### DISCUSSION THREAD

**Discussion by 996666925** [slint-ui/slint#4630] [Q&A]
How to embed slint rust into the qt window

**Comment by ogoffart** [comment]
We currently have API for that only if you're using the interpreter from C++, in that case, using the Qt backend, you can get the QWidget pointer qwidget() function. 
https://slint.dev/releases/1.4.1/docs/cpp/api/classslint_1_1interpreter_1_1ComponentInstance#_CPPv4NK5slint11interpreter17ComponentInstance7qwidgetEv
There is an example of that in https://github.com/slint-ui/slint/tree/master/examples/cpp/qt_viewer

What is your usecase? Are you using C++ or Rust?
